joomla/joomla-cms
# [4.0] Namespace com_contenthistory

Pull Request for Issue # .

### Summary of Changes
This simple PR converts com_contenthistory component to namespace MVC for Joomla 4.

### Testing Instructions
1. Install latest Joomla 4.0-dev with sample data, then apply patch (or better install my branch https://github.com/joomdonation/joomla-cms/archive/namespace_com_contenthistory.zip )

2. Login to administrator  Content -> Article, try to edit an article, save it (so that version of that version history of that article saved)

3. Try to edit that article again, click on Versions button, there will be a popup displays version history of that article. Check the Restore, Compare, Preview... , make sure it is still working as before

### Additional comment
When all components namespaced , we need to review and update data for #__content_types table, table field, data for prefix setting (currently, it is JTable, need to correct it to correct table prefix)
